Face-Detection/face_detect.py

```diff
@@ -0,0 +1,53 @@
+#Face and Eye Detection 
+import cv2
+
+#PATH of the FILE
+CLASSIFIER_PATH = "Facedetection/haarcascade_frontalface_default.xml"   
+EYEDETECTION_PATH = "Facedetection/haarcascade_eye.xml"
```

Comment:
```suggestion
EYEDETECTION_PATH = "./Facedetection/haarcascade_eye.xml"
```

Reply:
Sir I have made the changes, please check it.